### PR TITLE
fix: datamigration model validation

### PR DESCRIPTION
**Pull request by nschonni**
Remove empty body to fix RESPONSE_SCHEMA_NOT_IN_SPEC errors

### CODE REVIEW

**Review by lirenhe — APPROVED** [review]

**Comment by lirenhe** [comment]
I will wait for the review from code owner @hitenjava before merge the code

**Comment by lirenhe** [comment]
@hitenjava, could you take a look at this PR?

**Review by yungezz — APPROVED** [review]

**Pull request merged**
